### PR TITLE
Added series01 file for lesson3, assignment1

### PR DESCRIPTION
**Pull request by ptfbanks**
Previous submittal request failed.

### CODE REVIEW

File: students/ptfbanks/Lesson03/series01.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
+print ("This curious little routine plays games with strings and tuples, by parsing and splicing them in alernate ways.")
+print ("A string refers to an ordered sequence of characters; like a sentence.")
+print ("A tupple is an ordered sequence of entries (script limits 20 entries) delimited by commas.")
+print ("Words and variables may also be included as tupple elements when appropriate.")
+Sequence = []  #iniitalise sequence "Entry" Print ()
+print ()
+print ("Please enter a text string, or a series of tuple entry elements.")
+for i in range (20):
+    Add_entry = input("Enter string text or entry for tuple element Number.  (Enter END to stop) ")
```
Comment (natasha-aleksandrova) [inline students/ptfbanks/Lesson03/series01.py]: per pep8, var names should be snake_case -> `add_entry`

File: students/ptfbanks/Lesson03/series01.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    if Add_entry == "END":
+        break
+    Sequence.append(Add_entry)
+print ("Thank you.  The sequence entered is recorded as:")
+
+if i ==1:
+    Sequence = f"{Sequence}"
+    Sequence = Sequence [2:-2]
```
Comment (natasha-aleksandrova) [inline students/ptfbanks/Lesson03/series01.py]: again, lower snake_case and no spaces around brackets for slicing.

**Comment by ptfbanks** [comment]
Thank you.  I was not aware of snake_case nor that the convention was that
strongly held to.  I'll watch myself

**Comment by natasha-aleksandrova** [comment]
https://pep8.org/ is a pretty good read if you have some time! Most languages will have a linter to check for common style violations; this is something that is tracked as a metric in the industry to ensure production code is clean and consistent, especially when you're working with a large group of people on a team.

Here is a little more info on what type of code metrics are tracked in the industry:
https://www.fullstackpython.com/code-metrics.html

Getting used to python style guidelines is great way to start learning Python. I am also personally an advocate for using a really good IDE that auto-checks those for you as you write code 🙂 .

**Pull request merged**